packages/sol-cov/src/artifact_adapters/truffle.ts

```diff
@@ -0,0 +1,43 @@
+import { Compiler, CompilerOptions } from '@0xproject/sol-compiler';
+import * as fs from 'fs';
+import * as glob from 'glob';
+import * as _ from 'lodash';
+import * as path from 'path';
+import * as rimraf from 'rimraf';
+
+import { ContractData } from '../types';
+
+import { ZeroExArtifactAdapter } from './0x';
+import { AbstractArtifactAdapter } from './abstract';
+
+export class TruffleArtifactAdapter extends AbstractArtifactAdapter {
+    private _solcVersion: string;
+    private _sourcesPath: string;
+    constructor(sourcesPath: string, solcVersion: string) {
+        super();
+        this._solcVersion = solcVersion;
+        this._sourcesPath = sourcesPath;
+    }
+    public async collectContractsDataAsync(): Promise<ContractData[]> {
+        const artifactsDir = '0x-artifacts';
```

Comment:
Let's prefix this with a `.` so it's invisible to some people. Less scary when you see folders popup randomly when running code.